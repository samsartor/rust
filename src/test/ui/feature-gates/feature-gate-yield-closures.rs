fn main() {
    let _ = || { yield }; //~ ERROR yield syntax is experimental
}
