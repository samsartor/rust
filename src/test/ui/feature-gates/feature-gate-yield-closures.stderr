error[E0658]: yield syntax is experimental
  --> $DIR/feature-gate-yield-closures.rs:2:18
   |
LL |     let _ = || { yield };
   |                  ^^^^^
   |
   = note: see issue #43122 <https://github.com/rust-lang/rust/issues/43122> for more information
   = help: add `#![feature(generators)]` to the crate attributes to enable

error: aborting due to previous error

For more information about this error, try `rustc --explain E0658`.
